docs/source/contributing_playbooks.rst

```diff
@@ -0,0 +1,92 @@
+Contributing Playbooks
+----------------------
+
+Addition of new playbooks to the edpm-ansible collection is easy,
+thanks to specialized development tooling and CI pipeline.
+However, certain requirements must be met by any new playbook,
+in order to ensure quality of the collection and a smooth operation
+of software consuming it.
+
+These requirements, outlined in the following document, go beyond
+syntactic and functional correctness of the code.
+
+Where there isn't an explicit requirement, or recommendation,
+provided by this document, the primary `Ansible documentation`_ is considered
+to be the source of truth. When this document and official Ansible docs
+are in substantial conflict, an issue should be raised.
+
+Playbook design principles
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Playbooks tie multiple roles and tasks together into cohesive whole.
+If roles represent subroutines of Ansible world, playbooks are programs and scripts.
```

Comment:
I don't really agree with the analogy. Could we link to ansible docs describing playbooks instead of trying to do it ourselves? It seems redundant. I'd just link to https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_intro.html